src/Components/AllArticles.jsx

```diff
@@ -0,0 +1,28 @@
+import ArticleCard from "./ArticleCard";
+import { fetchAllArticles } from "./Utils/apis";
+import { useEffect, useState } from "react";
+
+
+
+export default function AllArticles() {
+
+    const [articles, setArticles] = useState([]);
```

Comment:
Please add a state for loading here and render a message for the user conditionally while the articles are loading.